docs/💖 Free Training/Alchemy's Road to Web3.md

```diff
@@ -0,0 +1,15 @@
+Learn about [Alchemy's Road to Web3 series](https://docs.alchemy.com/docs/welcome-to-the-road-to-web3), a 10-week, self-paced web3 developer program to help new blockchain devs go from beginner to advanced.
+
+## Road to Web3 Learning Challenges
```

Comment:
There's a missing space here (Markdown Linter).
```suggestion
## Road to Web3 Learning Challenges :rocket:

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable